packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -81,90 +68,125 @@ import {eventOptions} from 'lit/decorators/event-options.js';
-  ["@babel/plugin-proposal-decorators", {
-    "version": "2018-09",
-    "decoratorsBeforeExport": true
-  }],
-  ["@babel/plugin-proposal-class-properties"]
-]
+// tsconfig.json
+{
+  "compilerOptions": {
+    "experimentalDecorators": false, // default for TypeScript 5.0 and up
+    "useDefineForClassFields": true, // default when "target" is "ES2022" or higher
+  }
+}
 ```
 
-<div class="alert alert-info">
-
-Babel decorator support has been tested with `version: '2018-09'`. This is currently the default, but we recommend setting the version explicitly in case the default changes. Other versions ('2021-12' or 'legacy') are not supported, but this may change as Babel evolves. See the [Babel documentation](https://babeljs.io/docs/en/babel-plugin-proposal-decorators#options) if you want to experiment.
+Note: The `accessor` keyword was introduced in TypeScript 4.9 and full standard decorator with metadata support requires TypeScript 5.2 or greater.
 
-</div>
+### Using decorators with Babel { #decorators-babel }
 
-### Using decorators with TypeScript and Babel
+[Babel](https://babeljs.io/docs/en/) supports standard decorators with the [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators) plugin as of version 7.23. Babel does not support TypeScript experimental decorators, so you must use Lit decorators with [standard decorator syntax](#standard-decorators) using the `accessor` keyword on decorated class fields.
 
-When using TypeScript with Babel, it's important to order the TypeScript transform before the decorators transform in your Babel config as follows:
+Enable decorators by adding [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators) with these Babel configuration settings:
 
 ```json
+// babel.config.json
 {
-  "assumptions": {
-    "setPublicClassFields": true
-  },
   "plugins": [
-    ["@babel/plugin-transform-typescript", {
-      "allowDeclareFields": true
-    }],
-    ["@babel/plugin-proposal-decorators", {
-      "version": "2018-09",
-      "decoratorsBeforeExport": true
-    }],
-    ["@babel/plugin-proposal-class-properties"]
+    ["@babel/plugin-proposal-decorators", {"version": "2023-05"}]
   ]
 }
 ```
 
-The `allowDeclareFields` setting is generally not needed, but it can be useful if you want to define a reactive property without using a decorator. For example,
+Note: Lit decorators only work with `"version": "2023-05"`. Other versions, including the formerly supported `"2018-09"`, are not supported.
+
+### Avoiding issues with class fields and decorators {#avoiding-issues-with-class-fields}
+
+Standard [class fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields) have a problematic interaction with declaring reactive properties. See [Avoiding issues with class fields when declaring properties](/docs/v3/components/properties/#avoiding-issues-with-class-fields) for more information.
```

Comment:
That linked document doesn't mention the `accessor` keyword, which is our work around here.

I think this section is confusing here as written. Maybe we should just drop it, since its content is covered already?

Reply:
there's an issue for that here https://github.com/lit/lit.dev/issues/1243 which can be a follow up